core/src/main/java/tc/oc/pgm/doublejump/DoubleJumpMatchModule.java

```diff
@@ -65,6 +65,7 @@ public void disable() {
       iterator.remove();
       this.refreshJump(player);
     }
+    this.jumpers.disable();
```

Comment:
Unrelated bug fix?

Reply:
Yes, i was looking into using online map adapters, and saw this as an usage which was never disabled (so it's a memory leak)